darkcluster/src/test/java/com/linkedin/darkcluster/TestDarkClusterStrategyFactory.java

```diff
@@ -259,6 +311,18 @@ public void onClusterAdded(String clusterName)
       // deleting whatever the first cluster listener added. It would have been more straightforward to have
       // a handle directly to the other clusterListener, but there's no good reason for the StrategyFactory to
       // expose that or allow it to be passed in, as the clusterListener needs to manipulate internal state.
+      try
```

Comment:
this is not a new change, nothing has changed in this function. I don't know why github shows a diff here, the file looks fine.